server/manager/src/main/java/org/apache/accumulo/manager/metrics/fate/FateMetrics.java

```diff
@@ -149,6 +149,7 @@ public MicrometerFormatter(final MultiGauge mmFateMultiGauge) {
 
       for (ReadOnlyTStore.TStatus t : ReadOnlyTStore.TStatus.values()) {
         // types are from a fixed set
+        // this map is unused
         Map<String,Long> fateTypeCounts = new TreeMap<>();
```

Comment:
I noticed this map is being created but never used. Is there more planned for this?

Reply:
That might have been a place holder for being able to report values that returned to 0 and don't show up in the status.  Not sure if it is necessary anymore as long as the functionality is still present.

Say there is a fate type x.  While type x is running it will report the count.  When type x completes, w should still report type x, but with a 0 count.